### PR TITLE
Hello Microverse PR

### PR DESCRIPTION
**Pull request by Chandan-devs-tech**
What?

- I created the required files such as index.html and mystyle.css, .gitignore, test.md, and Set-up Github actions by adding a directory Github/workflows and linters.yml file inside it & other files associated including- .hintrc, .stylelintrc.json to meet the requirements of the project.
- I also changed what the reviewers required too, such as PR description and other optional changes.
Why?

- Due to project requirements and reviewer's ideas.
How?

- Files were created using git bash and edited using vscode and therefore pushed to the Github repository.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+header {
+  color: blue;
+}
```
Comment (bobb-Rob) [inline style.css:3]: Good job implementing the requirements outlined for this project. However, there are some issues you should take note of. 

- [ ] You did not follow the GitHub flow. All the core requirements for this project should have been implemented in the feature branch, then a pull request showing all things you implement is created. Please read the about [GitHub flow here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md). 
     👉 Kindly implement all the requirements in the feature branch, commit them and push. **_Please note that you do not need to open a new pull request, just push all your commits to this branch and then re-submit for review_**

- [ ] Pull Request(PR) should have a descriptive title describing the changes inside. Also, PR should have a summary describing introduced changes in more detail.  This is best practice as a developer, kindly update this pull request with a suitable title and description

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @Chandan-devs-tech , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Chandan-devs-tech — COMMENTED** [review]
The modification was done in the README.md file while adding a Slack account.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️

Hi @Chandan-devs-tech , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
✅ No Linter error
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages


 ## Required Changes ♻️
_Check the comments under the review._
- [REQUIRED ] Please note that the correct use of GitHub flow is fully implemented when all the changes are reflected in a pull request before it is merged with the master/main branch. I noticed that you have the project implementation in you master branch already. Please refer to my [previous review](https://github.com/Chandan-devs-tech/Hello-Microverse/pull/1/files#r955236178) and also read about [GitHub Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) here and implement accordingly.

     - One way to do this is to manually remove all the files in the master branch and feature branch in GitHub. Then, in your local repo, in the feature branch, push the files and it will reflect in this pull request.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Chandan-devs-tech — COMMENTED** [review]
Added mystyle.css  to the index.html file.

File: index.html [hunk 2]
```diff
@@ -5,7 +5,6 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>WELCOME</title>
-    <link rel="stylesheet" href="style.css">
```
Comment (ElsonOtake) [inline index.html:8]: - [ ] I think it's better to link your new `mystyle.css` file to `index.htm`. Your new styling does not work without this link.

**Review by ElsonOtake — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 

Hi @Chandan-devs-tech, 

Great work on making the changes requested by a previous reviewer 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🥲 🥲 

woohoo-youre-almost

 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] Please write short, detailed, easy to understand descriptions of introduced changes to your pull requests. It's a project requirement.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. 👍 👍 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Chandan-devs-tech) [reply index.html:8]: Yaa, I will link the mystyle.css file to index.html.

**Review by Chandan-devs-tech — COMMENTED** [review]
What?
- I created the required files such as index.html and mystyle.css, .gitignore, test.md, and Set-up Github actions by adding a directory Github/workflows and linters.yml file inside it & other files associated including- .hintrc, .stylelintrc.json to meet the requirements of the project.
- I also changed what the reviewers required too, such as PR description and other optional changes.

Why?
- Due to project requirements and reviewer's ideas.

How?
- Files were created using git bash and edited using vscode and therefore pushed to the Github repository.

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️
**Hi @Chandan-devs-tech,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 🎉


You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**